vm/src/builtins/float.rs

```diff
@@ -389,6 +389,16 @@ impl PyFloat {
         try_bigint(self.value, vm)
     }
 
+    #[pymethod(name = "__floor__")]
```

Comment:
you can replace these with `pymethod(magic)` and the name is handled automatically. I.e:

```suggestion
    #[pymethod(magic)]
```